packages/flutter/lib/src/material/drop_down.dart

```diff
@@ -493,18 +510,37 @@ class _DropDownButtonState<T> extends State<DropDownButton<T>> {
             alignment: FractionalOffset.centerLeft,
             children: config.items
           ),
-          new Icon(Icons.arrow_drop_down, size: config.iconSize)
+          new Icon(Icons.arrow_drop_down,
+            size: config.iconSize,
+            // These colors are not defined in the Material Design spec.
+            color: Theme.of(context).brightness == Brightness.light ? Colors.grey[700] : Colors.white70
+          )
         ]
       )
     );
+
     if (!DropDownButtonHideUnderline.at(context)) {
-      result = new Container(
-        decoration: const BoxDecoration(border: _kDropDownUnderline),
-        child: result
+      result = new Stack(
+        children: <Widget>[
+          result,
+          new Positioned(
+            left: 0.0,
+            right: 0.0,
+            bottom: 8.0,
```

Comment:
Why do this with a stack instead of a container with a bottom margin of 8.0 pixels and foregroundDecoration?